src/external/crypto.m4

```diff
@@ -5,17 +5,16 @@ AC_DEFUN([AM_CHECK_LIBCRYPTO],
 ])
 
 AC_MSG_CHECKING([whether OpenSSL's x400Address is ASN1_STRING])
-SAVE_CFLAGS=$CFLAGS
-CFLAGS="$CFLAGS -Werror -Wall -Wextra"
 AC_COMPILE_IFELSE(
                   [AC_LANG_SOURCE([
                       #include <openssl/x509v3.h>
 
                       int main(void)
                       {
                           GENERAL_NAME gn = { 0 };
-
-                          return ASN1_STRING_length(gn.d.x400Address);
+                          // If the types are different, it errors out.
```

Comment:
Hi,

please use `/* ... */` comments (https://sssd.io/contrib/coding-style.html). Additionally, `it` is imo a bit ambiguous here, better say `..., compiler errors out.`

bye,
Sumit